### ISSUE TITLE
warning 'same port for packet' suggestion

### ISSUE THREAD

**Issue by erickvermot** [kytos/kytos-napps#215]
While investigating issue https://github.com/kytos/kytos/issues/312 , I found a Q&A related to it: http://stackoverflow.com/questions/37134118/custom-mininet-topology-ping-error

Apparently, POX controller produces a Warning "Same port for packet", that helps identify issues related to arp.
Maybe Kytos could produce a similar warning.

I believe it is related to kytos/of_l2ls , but I might be wrong.

**Comment by diraol** [comment]
I have two comments regarding this issue (and the stackoverflow comment).

1) The given topology on SO have a loop. Currently we don't deal with topologies that contains loops.

2) As far as I remember, considering our `kytos/of_l2ls` implementation the given behavior will not happen, since we do not flood the `ARP Reply` (as described by the referenced SO comment) because we already know it's destination (because of the ARP Request).

**Comment by diraol** [comment]
I'll close this issue for now, as it is not based on a behavior that was effectively found on Kytos.

If, at some point, we identify this issue on kytos (and kytos/of_l2ls), then we can reopen this issue in order to evaluate how to deal with it.

**Comment by erickvermot** [comment]
This is not very important, but I'll comment here just to make things clear.

> The given topology on SO have a loop. Currently we don't deal with topologies that contains loops.

Yes, I know, but the topology on which I experienced it did not have a loop...

> As far as I remember, considering our kytos/of_l2ls implementation the given behavior will not happen, since we do not flood the ARP Reply (as described by the referenced SO comment) because we already know it's destination (because of the ARP Request).

If everything is fine with ARP that would be true, but I think that the point is that something was wrong with ARP, and the proposed warning would help realize it... Running mininet with --arp flag solved it. Maybe the problem was related to mininet's ARP, but this issue is not about saying it's kytos's fault, jut help the user realize what the problem is...

> I'll close this issue for now, as it is not based on a behavior that was effectively found on Kytos.
If, at some point, we identify this issue on kytos (and kytos/of_l2ls), then we can reopen this issue in order to evaluate how to deal with it.

I don't understand... It **was** effectively found on Kytos...
I only opened this improvement issue after talking to @beraldoleal about it. He agreed with implementing the warning and told me to open the issue, so I don't understand why close it...
Anyway, it is just an improvement...
